views/forms/UserProfileForm/UserProfileForm.test.jsx

```diff
@@ -66,15 +61,16 @@ describe('User Profile Form', () => {
     const props = { ...defaultProps, onSubmit }
 
     renderForm(props)
-    await user.type(elements.emailField(), 'not-an-email')
+    await user.type(elements.displayNameField(), 'My name')
+    await user.clear(elements.displayNameField())
     await user.click(elements.submitBtn())
 
-    await waitFor(() =>
-      expect(screen.getByText('mail must be a valid email')).toBeInTheDocument()
-    )
-    expect(
-      screen.getByText('display_name is a required field')
-    ).toBeInTheDocument()
+    await waitFor(() => {
+      expect(elements.emailField()).toBeDisabled()
+      expect(
+        screen.getByText('display_name is a required field')
+      ).toBeInTheDocument()
```

Comment:
there should only be 1 expectation in a `waitFor` function.

Reply:
[2b10634](https://github.com/AMP-SCZ/dpdash/pull/478/commits/2b10634c506de607903b3c18f8cbfd56e8904186)